ccp/src/cu/proving_thread/sync/channels_facade.rs

```diff
@@ -69,24 +71,19 @@ impl ToAsync {
 }
 
 #[derive(Clone, Debug)]
-pub(crate) struct ToUtility {
-    to_utility: ToUtilityInlet,
+pub(crate) struct ToStateOperating {
+    to_state_operating: ToStateOperatingInlet,
 }
 
-impl ToUtility {
-    pub(crate) fn new(to_utility: ToUtilityInlet) -> Self {
-        Self { to_utility }
+impl ToStateOperating {
```

Comment:
let's move this definition right after the `ToAsync` deifintion, so it will look like:
```
pub(crate) struct ToAsync { ... }

pub(crate) struct ToStateOperating { ... }

impl ToStateOperating { ... }

impl ToStatsCollecting { ... }
```